2019/python/lar_constraints.py

```diff
@@ -452,11 +452,11 @@ def v644_const(self, row):
 		"""1) If Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname equals 2,
 			then Sex of Applicant or Borrower must equal 1, 2, 3, or 6.
 		2) If Sex of Applicant or Borrower equals 6, then Sex of Applicant or Borrower Collected on the Basis of
-			Visual Observation or Surname must equal 2."""
+			Visual Observation or Surname must equal 2 or 3."""
 		if row["app_sex_basis"] == "2" and row["app_sex"] not in ("1", "2", "3", "6"):
 			row["app_sex"] = random.choice(("1", "2", "3", "6"))
 		if row["app_sex"] == "6":
-			row["app_sex_basis"] = "2"
+			row["app_sex_basis"] random.choice("2", "3")
```

Comment:
I only see a 2 listed in https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2019-hmda-fig.pdf.

Can you confirm the source of 2 and 3?

Reply:
This change is reflected in the last commit.